io.github.eteran.edb-debugger.yml

```diff
@@ -0,0 +1,78 @@
+app-id: io.github.eteran.edb-debugger
+runtime: org.kde.Platform
+runtime-version: '5.15-23.08'
+sdk: org.kde.Sdk
+command: edb
+rename-icon: edb
+rename-desktop-file: edb.desktop
+rename-appdata-file: edb.appdata.xml
+finish-args:
+  - --allow=devel
+  - --share=ipc
+  - --socket=x11
+  - --filesystem=xdg-documents
+  - --device=dri
+
+modules:
+  - name: capstone
+    buildsystem: simple
+    build-commands:
+      - make V=1 PREFIX=/app/
+      - make DESTDIR=/app install
+      - mkdir /app/lib /app/include /app/bin
+      - mv /app/usr/lib/* /app/lib/
+      - mv /app/usr/include/* /app/include/
+      - mv /app/usr/bin/* /app/bin/
+    post-install:
+      - rm -r /app/usr
+    sources:
+      - type: git
+        url: https://github.com/aquynh/capstone.git
+        tag: "5.0.1"
+        commit: 097c04d9413c59a58b00d4d1c8d5dc0ac158ffaa
+
+  - name: libXmu-xterm
+    buildsystem: autotools
+    sources:
+      - type: archive
+        url: https://xorg.freedesktop.org/releases/individual/lib/libXmu-1.1.4.tar.gz
+        sha256: 3091d711cdc1d8ea0f545a13b90d1464c3c3ab64778fd121f0d789b277a80289
+
+  - name: libXaw-xterm
+    buildsystem: autotools
+    sources:
+      - type: archive
+        url: https://xorg.freedesktop.org/releases/individual/lib/libXaw-1.0.16.tar.gz
+        sha256: 012f90adf8739f2f023d63a5fee1528949cf2aba92ef7ac1abcfc2ae9cf28798
+
+  - name: xterm
+    buildsystem: autotools
+    config-opts:
+      - --disable-full-tgetent
+    sources:
+      - type: archive
+        url: https://invisible-island.net/datafiles/release/xterm.tar.gz
+        sha256: 75117c3cc5174a09c425ef106e69404d72f5ef05e03a5da00aaf15792d6f9c0f
+
+  - name: graphviz
+    buildsystem: autotools
+    sources:
+      - type: archive
+        url: https://gitlab.com/api/v4/projects/4207231/packages/generic/graphviz-releases/10.0.1/graphviz-10.0.1.tar.gz
+        sha256: 72e1a575e9c382c4fd2267837962bc4582b3d9ad746e338cd8024bdeca124f32
+
+  - name: edb
+    buildsystem: cmake-ninja
+    post-install:
+      #xterm path
+      - sed -i "s/\x2f\x75\x73\x72\x2f\x62\x69\x6e\x2f\x78\x74\x65\x72\x6d/\x78\x74\x65\x72\x6d\x00\x00\x00\x00\x00\x00\x00\x00\x00/g" /app/bin/edb
```

Comment:
DO NOT BINARY patch like that. Patch the source code.

Reply:
I would prefer to leave the line because it is easier to maintain. If you have to patch, I think look, they wanted two files. It is quite unlikely that the line will end in the future causing a failure because it is very specific. (this is a translation)

Reply:
no it is not easier to maintain. It is completely obscure and it will break randomly as sed failure mode is inexistant.